bitcoin/rpc.py

```diff
@@ -10,7 +10,8 @@
 # propagated, or distributed except according to the terms contained in the
 # LICENSE file.
 
-"""Bitcoin Core RPC support"""
+"""Bitcoin Core RPC support. By default this uses the standard library ``json``
```

Comment:
Make "By default" on a separate line, with a space between it and the title  "Bitcoin Core RPC support"